InSAR/multiburst/multiburst_insar.ipynb

```diff
@@ -0,0 +1,261 @@
+   "id": "2bd6ce83-e6a9-4953-a911-c606efdf194c",
+   "metadata": {},
+   "source": [
+    "![Top SLC InSAR Pair](./assets/top_slc.png \"Top SLC InSAR Pair\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5ec0e52e-6516-474f-aeaa-22a1f09a85f7",
+   "metadata": {},
+   "source": [
+    "## Lower SLC Processing\n",
+    "\n",
+    "<a href=\"https://search.asf.alaska.edu/#/?maxResults=250&zoom=7.504&center=-114.798,33.203&polygon=POLYGON((-117.8581%2035.2613,-117.0303%2035.2613,-117.0303%2035.9371,-117.8581%2035.9371,-117.8581%2035.2613))&searchType=Geographic%20Search&resultsLoaded=true&granule=S1A_IW_SLC__1SDV_20190704T015023_20190704T015051_027961_03283A_D6EA-SLC&productTypes=SLC&flightDirs=Ascending&start=2019-07-03T05:00:00Z&end=2019-07-25T04:59:59Z&path=64-64&frame=109-109\" > View the input Sentinel-1 scenes in Vertex. </a>\n",
+    "\n",
+    "The command below takes about 30 minutes to run when the call to `hyp3_isce2` is uncommented."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e272a0e2-76a3-4f50-80ed-74a2df2cd9c6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " 21G\t/Users/ffwilliams2/Repositories/hyp3/tutorials/InSAR/multiburst/slc_lower\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash -s \"$slc_lower\"\n",
+    "cd $1\n",
+    "# python -m hyp3_isce2 ++process insar_tops \\\n",
+    "#     --reference S1A_IW_SLC__1SDV_20190704T015023_20190704T015051_027961_03283A_D6EA \\\n",
+    "#     --secondary S1B_IW_SLC__1SDV_20190710T014959_20190710T015026_017065_0201B8_069B\n",
+    "du -sh $1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44c91659-0072-45d8-9639-70ef9440247a",
+   "metadata": {},
+   "source": [
+    "![Bottom SLC InSAR Pair](./assets/bottom_slc.png \"Bottom SLC InSAR Pair\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68c40dcb-1037-444e-b620-fb17b7c7998c",
+   "metadata": {},
+   "source": [
+    "From here, you'd have to merge the two InSAR pairs, which is very challenging due to differences in the reference point and unwrapping decisions made for each pair. The lack of the ability to merge InSAR pairs after processing is a drawback of current SLC-based HyP3 InSAR job."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd185a0d-459d-4ca0-aead-0498ce3164a3",
+   "metadata": {},
+   "source": [
+    "## Multiburst Processing\n",
+    "\n",
+    "<a href=\"https://search.asf.alaska.edu/#/?maxResults=250&zoom=8.511&center=-118.003,34.754&polygon=POLYGON((-117.5715%2035.3207,-117.1319%2035.3207,-117.1319%2035.9412,-117.5715%2035.9412,-117.5715%2035.3207))&searchType=Geographic%20Search&resultsLoaded=true&granule=S1_135528_IW2_20190716T015049_VV_6DF2-BURST&path=64-64&frame=109-109&flightDirs=Ascending&start=2019-07-15T05:00:00Z&end=2019-07-18T04:59:59Z&dataset=SENTINEL-1%20BURSTS&polarizations=VV\" > View input Sentinel-1 scenes in Vertex. </a>\n",
```

Comment:
This pulls back a set of 10 bursts from 2019-07-16, not the 20 bursts being used in the example. Updating the start/end dates to 2024-07-04 and 2024-07-11 does the trick:

```suggestion
    "<a href=\"https://search.asf.alaska.edu/#/?maxResults=250&zoom=8.511&center=-118.003,34.754&polygon=POLYGON((-117.5715%2035.3207,-117.1319%2035.3207,-117.1319%2035.9412,-117.5715%2035.9412,-117.5715%2035.3207))&searchType=Geographic%20Search&resultsLoaded=true&granule=S1_135528_IW2_20190716T015049_VV_6DF2-BURST&path=64-64&frame=109-109&flightDirs=Ascending&start=2019-07-04T00:00:00Z&end=2019-07-11T00:00:00Z&dataset=SENTINEL-1%20BURSTS&polarizations=VV\" > View input Sentinel-1 scenes in Vertex. </a>\n",
```